testsuite/performance/tests/pom.xml

```diff
@@ -48,11 +48,25 @@
         <keycloak.ds.max-pool-size>100</keycloak.ds.max-pool-size>
         <keycloak.ds.pool-prefill>true</keycloak.ds.pool-prefill>
         <keycloak.ds.ps-cache-size>100</keycloak.ds.ps-cache-size>
-        
+
         <keycloak-lb.jvm.memory>-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m</keycloak-lb.jvm.memory>
         <keycloak-lb.http.max-connections>500</keycloak-lb.http.max-connections>
         <keycloak-lb.worker.io-threads>2</keycloak-lb.worker.io-threads>
         <keycloak-lb.worker.task-max-threads>16</keycloak-lb.worker.task-max-threads>
+
+        <docker.db.cpuset>0-1</docker.db.cpuset>
+        <docker.keycloak.cpuset>2-3</docker.keycloak.cpuset>
```

Comment:
Does this mean the tests will require minimum 4 CPU cores? We should add that info to the README requirements section.

Reply:
Thanks, added.